New Line_Flowering Parameter

wutermeloon opened the pull request:
Some new parameters have been added, stop_at_end and min_speed. Also, some parameters have been given default values so that they don't take up as much space.

wutermeloon committed:
New Line_Flowering Parameter

Some new parameters have been added, stop_at_end and min_speed. Also, some parameters have been given default values so that they don't take up as much space.

wutermeloon committed:
First batch of missions! :D

The traffic_crane_mission is the program you have seen befor to do the traffice mission and the crane mission. The swing2ramp mission is for the swing, support structure, elevator, and ramp missions. You've seen the push_blocks mission before.

Butterspot commented on Traffic_Crane.py:
```diff
@@ -0,0 +1,46 @@
+    #Turning towards crane mission
+    m.on(30, -30)
+    #Stoping at line connected to base
+    while is_left_black() != True:
```
Can we change this to "is_left_not_black" to make it simpler?

wutermeloon replied:
I have changed the "not_white" to "other_shade" to say it is not black or white.

Butterspot commented on color.py:
```diff
@@ -29,7 +29,7 @@ def is_left_black():
         return False
 
 def is_left_not_white():
-    if left_color_sensor_rli() < 90:
```
If the reflection value is 50, should it be not black, or not white?

wutermeloon replied:
The value 50 in this program area is to determine that the value is not white and is most likely the other colors on the map.

wutermeloon committed:
Changed Name

Changed the name of "is_right_not_white" and "is_left_not_white" to "is_<xyz>_other_shade

wutermeloon committed:
Main Program

This is a cambination of each of our mission programs put into one program. Instead of using a third color sensor, we're using buttons to control the order that we run the missions in. The middle button will call the missions one at a time in order instead of only controling one mission.

wutermeloon committed:
Mission Improvments

Fixed some turning problems with the pushing blocks program and used sideways gyro sensor to calculate when we have reached the top of the ramp

wutermeloon committed:
Improvents to reliability

Changed & programmed the treehouse mission to follow the circle-ish line to the tree

wutermeloon committed:
Changed mission order

Instead of going to the crane and then backing onto the circle-ish line to get to the tree, we will go to the tree and then go straight to the crane.

wutermeloon committed:
Speed upgrade

Made the missions faster in places that we could to save time since we are a little over 2:30 minutes.
